kernel/latest/default.nix

```diff
@@ -49,19 +49,17 @@ linux_latest.override({
 
     # Misc. community patches
     # None are *required* for basic function.
-    # https://github.com/nadiaholmquist/pbp-packages/tree/master/linux-pbp
-    (nhp "0007-mmc-core-pwrseq_simple-disable-mmc-power-on-shutdown.patch"         "1d16gjgds670dzpkb8jjlymcpp1inab3mlzbpfdinrgvfy4pywhi")
```

Comment:
This was needed to fix an issue with Wi-Fi at reboot.

 - https://github.com/torvalds/linux/blob/master/drivers/mmc/core/pwrseq_simple.c

Upstream doesn't have an equivalent change, but that doesn't mean it wasn't fixed elsewhere.

(If there was a centralized distro-agnostic repository of information about vendor devices we wouldn't need to second-guess everything :()

Reply:
Not sure what the original issue was but wifi works after reboot for me without this patch.